ToolBox/Program.cs

```diff
@@ -0,0 +1,190 @@
+                        break;
+                    case "asoc":
+                    case "algoseekoptionsconverter":
+                        AlgoSeekOptionsConverterProgram.AlgoSeekOptionsConverter(GetParameterOrExit(optionsObject, "date"));
+                        break;
+                    case "ivec":
+                    case "ivolatilityequityconverter":
+                        IVolatilityEquityConverterProgram.IVolatilityEquityConverter(GetParameterOrExit(optionsObject, "source-dir"),
+                                                                                     GetParameterOrExit(optionsObject, "source-meta-dir"),
+                                                                                     GetParameterOrExit(optionsObject, "destination-dir"),
+                                                                                     GetParameterOrExit(optionsObject, "resolution"));
+                        break;
+                    case "kdc":
+                    case "kaikodataconverter":
+                        KaikoDataConverterProgram.KaikoDataConverter(GetParameterOrExit(optionsObject, "market"),
+                                                                     GetParameterOrExit(optionsObject, "tick-type"),
+                                                                     GetParameterOrExit(optionsObject, "source-dir"));
+                        break;
+                    case "nmdc":
+                    case "nsemarketdataconverter":
+                        NseMarketDataConverterProgram.NseMarketDataConverter(GetParameterOrExit(optionsObject, "source-dir"),
+                                                                             GetParameterOrExit(optionsObject, "destination-dir"));
+                        break;
+                    case "qqc":
+                    case "quantquoteconverter":
+                        QuantQuoteConverterProgram.QuantQuoteConverter(GetParameterOrExit(optionsObject, "destination-dir"),
+                                                                       GetParameterOrExit(optionsObject, "source-dir"),
+                                                                       GetParameterOrExit(optionsObject, "resolution"));
+                        break;
+                    case "cug":
+                    case "coarseuniversegenerator":
+                        CoarseUniverseGeneratorProgram.CoarseUniverseGenerator();
+                        break;
+                    default:
+                        PrintMessageAndExit("ERROR: Unrecognized --app value");
+                        break;
+                }
+            }
+        }
+        private static IList<string> DictionaryObjectToList(object dictionaryObject)
+        {
+            IList<string> list = new List<string>();
+            var dictionary = dictionaryObject as Dictionary<string, object>;
+            foreach (var item in dictionary)
```

Comment:
no change required, but perhaps much more simply, `return dictionary.Keys.ToList()`

Reply:
Your right! I'll just lose this method and use `dictionary.Keys.ToList()` approach